Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean

```diff
@@ -638,14 +637,14 @@ protected theorem IsEmbedding.measurableEmbedding {f : α → β} (h₁ : IsEmbe
 @[deprecated (since := "2024-10-26")]
 alias Embedding.measurableEmbedding := IsEmbedding.measurableEmbedding
 
-protected theorem IsClosedEmbedding.measurableEmbedding {f : α → β}
+protected theorem Topology.IsClosedEmbedding.measurableEmbedding {f : α → β}
     (h : IsClosedEmbedding f) : MeasurableEmbedding f :=
   h.isEmbedding.measurableEmbedding h.isClosed_range.measurableSet
 
 @[deprecated (since := "2024-10-20")]
 alias ClosedEmbedding.measurableEmbedding := IsClosedEmbedding.measurableEmbedding
```

Comment:
Do we now need another `alias` for `IsClosedEmbedding.measurableEmbedding` → `Topology.IsClosedEmbedding.measurableEmbedding`? And similarly everywhere else. Or do you claim that the renames are close enough together in time that we only need the one already present?

Reply:
It is really really timetaking to write deprecated aliases manually and in that case the renames were indeed pretty close in time, so I'm of the opinion to stick to whatever `scripts/add_deprecations.sh` generated